README.md

```diff
@@ -12,6 +12,29 @@ OSV-Scanner provides an officially supported frontend to the [OSV database](http
 
 The above all results in fewer, more actionable vulnerability notifications, which reduces the time needed to resolve them.
 
+## Table of Contents
+- [OSV-Scanner](#osv-scanner)
+  - [Table of Contents](#table-of-contents)
+  - [Installing](#installing)
+    - [Installation Process](#installation-process)
+      - [Install from source](#install-from-source)
+    - [SemVer Adherence](#semver-adherence)
+  - [Usage](#usage)
+    - [Scan a directory](#scan-a-directory)
+      - [Example](#example)
```

Comment:
These examples make the ToC look a bit too dense. Should we omit these?

Reply:
I set the max indentation to 3 now, which removes the examples.